bin/homeshick

```diff
@@ -171,6 +171,7 @@ case $cmd in
   cd)    help cd        ;; # cd is implemented in the homeshick.{sh,csh} helper script.
   help)  help $help_cmd ;;
   *)
+    declare -a pull_completed
```

Comment:
Not that this is wrong, but the rest of the codebase uses `local pull_completed=()`. Could you change this just to keep it consistent?